src/v2/Apps/Show/ShowApp.tsx

```diff
@@ -28,26 +27,19 @@ import {
 } from "v2/Components/ArtworkFilter/ArtworkFilterContext"
 import { RouterLink } from "v2/System/Router/RouterLink"
 import { ShowNavigationBannerFragmentContainer as ShowNavigationBanner } from "./Components/ShowNavigationBanner"
-import { useRouter } from "v2/System/Router/useRouter"
 
 interface ShowAppProps {
   show: ShowApp_show
 }
 
 export const ShowApp: React.FC<ShowAppProps> = ({ show }) => {
   const { contextPageOwnerSlug, contextPageOwnerType } = useAnalyticsContext()
-  const { match } = useRouter()
-  const { from_fair } = match.location.query
-
-  // FIXME: Why is this memoized?
-  // eslint-disable-next-line react-hooks/exhaustive-deps
-  const arrivedFromFair = useMemo(() => from_fair ?? false, [])
```

Comment:
🙏

Reply:
unfortunately, we will still need to use this code 😅